lib/Doctrine/ODM/MongoDB/Aggregation/Expr.php

```diff
@@ -1600,7 +1600,9 @@ private function requiresCurrentField(?string $method = null): void
         }
     }
 
-    /** @throws BadMethodCallException If there is no current switch operator. */
+    /**
+     * @throws BadMethodCallException If there is no current switch operator.
+     */
```

Comment:
CI is complaining about this change, it needs to be reverted back:
```suggestion
    /** @throws BadMethodCallException If there is no current switch operator. */
```